docs/onboard-to-seed.md

```diff
@@ -0,0 +1,54 @@
+# Request for SEED and submit Intune device ID for SEED onboarding
+
+This article is applicable only for public officers who have an existing active TechPass ID and need to onboard their device to SEED. Public officers can request for SEED from the [TechPass portal](https://portal.techpass.gov.sg/public/home) using their non-SE GSIB machine.
+
+>**Notes**:
+>- You can't onboard a GSIB or DWP device to SEED.
+>- If you are a vendor or contractor, contact your project manager or reporting officer to request for SEED provisioning before proceeding with your SEED onboarding.
+
+
+_To onboard your device to SEED:_
+
+1. Go to [TechPass portal](https://portal.techpass.gov.sg) and click **Login with TechPass**.
+
+<kbd>![log in](assets/images/onboarding/po-non-se/log-in-with-techpass.png ':size=90%')</kbd>
+
+2. Choose your TechPass account.
```

Comment:
```suggestion
2. Sign In / Choose your TechPass account.
```

Reply:
noted, i have used "Sign in to your TechPass account."